src/scripting_api/field.js

```diff
@@ -188,6 +188,18 @@ class Field extends PDFObject {
     throw new Error("field.page is read-only");
   }
 
+  get rotation() {
+    return this._rotation;
+  }
+
+  set rotation(angle) {
+    angle = Math.floor(angle);
```

Comment:
Is it *correct* to accept a non-integer angle here?

Reply:
Yes it's: I checked in Acrobat.